osu.Game.Tests/Visual/Multiplayer/TestSceneDrawableRoomPlaylist.cs

```diff
@@ -128,7 +128,8 @@ public void TestEditableSelectable()
                 p.AllowSelection = true;
             });
 
-            moveToItem(0);
+            // ModIcon will handle clicks before EditableSelectable
+            moveToItem(0, new Vector2(50, 0));
```

Comment:
This is not an acceptable way to fix any tests and precisely demonstrates why setting `ModIcon.Action` was as bad an idea as I'm saying it was.